Formula/sslyze.rb

```diff
@@ -32,10 +32,11 @@ class Sslyze < Formula
 
   depends_on "pyinvoke" => :build
   depends_on "rust" => :build # for cryptography
-  depends_on arch: :x86_64 # https://github.com/nabla-c0d3/nassl/issues/83
   depends_on "openssl@1.1"
+  depends_on "pycparser"
```

Comment:
Should this be added to `pypi_formula_mappings.json`?

Reply:
the pypi actually does not work for this formula. (as it is not straightforward formula version bump)